bundles/org.openhab.binding.bluelink/src/main/resources/OH-INF/thing/vehicle.xml

```diff
@@ -0,0 +1,308 @@
+			<channel id="locked" typeId="locked"/>
+			<channel id="engine-running" typeId="engine-running"/>
+			<channel id="odometer" typeId="odometer"/>
+			<channel id="battery-level" typeId="battery-level"/>
+			<channel id="location" typeId="system.location"/>
+			<channel id="last-update" typeId="last-update"/>
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="charging">
+		<label>Charging</label>
+	</channel-group-type>
+
+	<channel-group-type id="fuel">
+		<label>Fuel</label>
+	</channel-group-type>
+
+	<channel-group-type id="doors">
+		<label>Doors</label>
+		<channels>
+			<channel id="front-left" typeId="door-state"/>
+			<channel id="front-right" typeId="door-state"/>
+			<channel id="rear-left" typeId="door-state"/>
+			<channel id="rear-right" typeId="door-state"/>
+			<channel id="trunk" typeId="trunk-state"/>
+			<channel id="hood" typeId="hood-state"/>
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="windows">
+		<label>Windows</label>
+		<channels>
+			<channel id="front-left" typeId="window-state"/>
+			<channel id="front-right" typeId="window-state"/>
+			<channel id="rear-left" typeId="window-state"/>
+			<channel id="rear-right" typeId="window-state"/>
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="climate">
+		<label>Climate Control</label>
+		<channels>
+			<channel id="hvac-on" typeId="hvac-on"/>
+			<channel id="temperature" typeId="temperature"/>
+			<channel id="defrost" typeId="defrost"/>
+			<channel id="seat-front-left" typeId="seat-heater"/>
+			<channel id="seat-front-right" typeId="seat-heater"/>
+			<channel id="seat-rear-left" typeId="seat-heater"/>
+			<channel id="seat-rear-right" typeId="seat-heater"/>
+			<channel id="steering-heater" typeId="heater-state"/>
+			<channel id="rear-window-heater" typeId="heater-state"/>
+			<channel id="side-mirror-heater" typeId="heater-state"/>
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="range">
+		<label>Driving Range</label>
+		<channels>
+			<channel id="total-range" typeId="range"/>
+			<channel id="ev-range" typeId="range"/>
+			<channel id="fuel-range" typeId="range"/>
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="warnings">
+		<label>Warnings</label>
+		<channels>
+			<channel id="tire-pressure" typeId="warning"/>
+			<channel id="tire-pressure-front-right" typeId="warning"/>
+			<channel id="tire-pressure-front-left" typeId="warning"/>
+			<channel id="tire-pressure-rear-right" typeId="warning"/>
+			<channel id="tire-pressure-rear-left" typeId="warning"/>
+			<channel id="washer-fluid" typeId="warning"/>
+			<channel id="brake-fluid" typeId="warning"/>
+			<channel id="smart-key-battery" typeId="warning"/>
+		</channels>
+	</channel-group-type>
+
+	<!-- Channel Types -->
+
+	<channel-type id="locked">
+		<item-type>Switch</item-type>
+		<label>Locked</label>
+		<description>Vehicle door lock status. Send ON to lock, OFF to unlock.</description>
+		<category>Lock</category>
```

Comment:
Can you add semantic tags (Point+ Property) where possible to all channel types.